### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by akshay-ap**
Related to #27

Changes in PR:
- Move package info like name, version, etc. to top
- Add homepage url
- Update repository url
- Update version to v0.1.0-alpha.2
- Rearrange import
- update script `postinstall`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +28,7 @@
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
     "lint:ts": "eslint 'test/**/*.ts' --max-warnings 0 --fix",
-    "typechain": "TS_NODE_TRANSPILE_ONLY=true hardhat typechain",
+    "typechain": "TS_NODE_TRANSPILE_ONLY=true yarn hardhat typechain",
```
Comment (mmv08) [inline package.json:31]: Why is this required? Since hardhat is a dependency it shouldn't be necessary

**Review by mmv08 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (akshay-ap) [reply package.json:31]: getting the error below when I try to install package demo repo:
Screenshot 2023-07-13 at 18 05 53

Comment (akshay-ap) [reply package.json:31]: and if hardhat is installed globally, postinstall still fails.
